packages/components/src/experience-control/index.stories.tsx

```diff
@@ -12,33 +13,39 @@ type Story = StoryObj< typeof ExperienceControl >;
 export const Default: Story = {
 	args: {
 		label: 'How was your experience?',
-		selectedExperience: 'good',
-		onChange: () => {},
+		value: ExperienceType.GOOD,
+		onChange: fn(),
 	},
 };
 
 export const WithHelpText: Story = {
 	args: {
 		label: 'Rate your satisfaction',
-		selectedExperience: 'good',
-		helpText: 'Please select an option that best describes your experience',
-		onChange: () => {},
+		value: ExperienceType.GOOD,
+		help: 'Please select an option that best describes your experience',
+		onChange: fn(),
 	},
 };
 
 export const PreSelectedBad: Story = {
 	args: {
 		label: 'How was the support?',
-		selectedExperience: 'bad',
-		onChange: () => {},
+		value: ExperienceType.BAD,
+		onChange: fn(),
 	},
 };
 
 // Example of using the Base component directly
 export const CustomBase: Story = {
 	render: () => (
 		<ExperienceControl.Base label="Custom Experience Control">
-			<ExperienceControl.Option isSelected onClick={ () => {} }>
+			<ExperienceControl.Option
+				checked
+				onClick={ fn() }
+				value="1"
+				name="experience-control"
+				ariaLabel="Rate as good experience"
+			>
 				1
 			</ExperienceControl.Option>
 		</ExperienceControl.Base>
```

Comment:
Let's remove this story for now, and we can come back to custom examples when we're ready?